src/components/Header/index.jsx

```diff
@@ -1,12 +1,13 @@
 import React from 'react';
-import ContactTop from '../ContactTop';
+// import ContactTop from '../ContactTop';
+import { ContactTopContainer } from '../../otherPagesComponent/ContactTop';
```

Comment:
there is no need to create `otherPagesComponent` you should create them directly in `components`